cmd/kube-rbac-proxy/app/kube-rbac-proxy.go

```diff
@@ -303,25 +287,26 @@ func secureServerRunner(
 	return runner, interrupter
 }
 
-func setupAuthorizer(krbInfo *server.KubeRBACProxyInfo, delegatedAuthz *serverconfig.AuthorizationInfo) (authorizer.Authorizer, error) {
-	staticAuthorizer, err := static.NewStaticAuthorizer(krbInfo.Authorization.Static)
-	if err != nil {
-		return nil, fmt.Errorf("failed to create static authorizer: %w", err)
+func setupAuthorizer(krpInfo *server.KubeRBACProxyInfo, delegatedAuthz *serverconfig.AuthorizationInfo) (authorizer.Authorizer, error) {
+	authz := delegatedAuthz.Authorizer
+
+	if len(krpInfo.Authorization.Static) > 0 {
+		staticAuthorizer, err := static.NewStaticAuthorizer(krpInfo.Authorization.Static)
+		if err != nil {
+			return nil, fmt.Errorf("failed to create static authorizer: %w", err)
+		}
+		authz = union.New(staticAuthorizer, authz)
 	}
 
-	var authz authorizer.Authorizer = rewrite.NewRewritingAuthorizer(
-		union.New(
-			staticAuthorizer,
-			delegatedAuthz.Authorizer,
-		),
-		krbInfo.Authorization.RewriteAttributesConfig,
-	)
+	if krpInfo.Authorization.RewriteAttributesConfig != nil {
```

Comment:
@ibihim I've added a unit test to verify that when not configured explicitly, `RewriteAttributesConfig` is indeed nil; this is the case because [unmarshalling a yaml config file](https://github.com/brancz/kube-rbac-proxy/blob/sig-auth-acceptance/cmd/kube-rbac-proxy/app/options/proxyoptions.go#L147) that doesn't set `RewriteAttributesConfig` will set the field to the zero value which is nil.